src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -218,8 +228,7 @@ public String getRawKeyName() {
      * @param vc          the variant context to annotate
      * @param likelihoods likelihoods indexed by sample, allele, and read within sample
      */
-    //@Override
-    public Map<String, Object> annotateRawData(ReferenceContext ref, VariantContext vc, AlleleLikelihoods<GATKRead, Allele> likelihoods) {
+    public static Map<String, Object> annotateRawData(ReferenceContext ref, VariantContext vc, AlleleLikelihoods<GATKRead, Allele> likelihoods) {
```

Comment:
I'm not sure why this doesn't already implement `ReducibleAnnotation`, but that's why there used to be an `@Override`.  Can you check to see what happens if you add that interface?  It's not unlikely there's a reason this is the exception, but I'd like to remember what it is and document it.

Reply:
Ah, thanks for that clarification! Hmm, looks like it's missing `getPrimaryRawKey`, `getSecondaryRawKeys`, and `hasSecondaryRawKeys` methods. Not sure what exactly these are supposed to return, but happy to add them if you can enlighten me!

Reply:
OK, I've reverted these changes and filed https://github.com/broadinstitute/gatk/issues/7564.